src/stylesheets/components/_icon-list.scss

```diff
@@ -0,0 +1,152 @@
+          .usa-icon-list__icon {
+            color: color($prefix);
+          }
+        }
+      }
+    }
+  }
+}
+
+.usa-icon-list__icon {
+  .usa-icon {
+    // Set the height and width of the icon based on the size variant and factor
+    display: block;
+    height: $type-scale * $icon-list-icon-size-factor;
+    margin-top: $icon-list-icon-margin-top;
+    position: relative;
+    width: $type-scale * $icon-list-icon-size-factor;
+  }
+}
+
+// Generate responsive variants
+// Create a map for looping that includes a default or no-breakpoint version
+$this-null: (
+  "none": null,
+);
+$icon-list-breakpoints: map-merge($this-null, $system-breakpoints);
+@each $mq-key, $mq-value in $icon-list-breakpoints {
+  @each $token, $val in $icon-list-font-sizes {
+    // Set the prefix
+    // An empty string if "none"
+    $prefix: false;
+    @if $mq-key == "none" {
+      $prefix: "";
+    }
+    // Or the standard prefix if the breakpoint is output
+    @else if map-get($theme-utility-breakpoints, $mq-key) {
+      $prefix: "#{$mq-key}\\:";
+    }
+    @if $prefix {
+      $this-type-scale: font-size($theme-icon-list-font-family, $token);
+      @include at-media($mq-key) {
+        .#{$prefix}usa-icon-list--size-#{$token} {
+          .usa-icon-list__icon {
+            .usa-icon {
+              // Set the height and width of the icon based on the size variant and factor
+              height: $this-type-scale * $icon-list-icon-size-factor;
+              margin-top: $icon-list-icon-margin-top;
+              width: $this-type-scale * $icon-list-icon-size-factor;
+            }
+          }
+
+          .usa-icon-list__content {
+            // Resize simple (un-marked up) content
+            font-size: size($theme-icon-list-font-family, $token);
+            // Calculate the space between the icon and content based on the size variant and factor
+            padding-left: $this-type-scale * $icon-list-icon-padding-left-factor;
+
+            // But don't resize any block-level element inside content
+            > p,
+            > h2,
+            > h3,
+            > h4,
+            > h5,
+            > h6,
+            > ul,
+            > ol,
+            > div {
+              font-size: initial;
+            }
+
+            .usa-icon-list__title {
+              font-size: size($theme-icon-list-font-family, $token);
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+.usa-icon-list__title {
+  font-size: size($theme-icon-list-font-family, $theme-body-font-size);
+  line-height: lh($theme-icon-list-font-family, $theme-heading-line-height);
+  margin-bottom: units(0);
+  padding-top: $icon-list-title-padding-top;
+  & + * {
+    margin-top: units(1);
+  }
+}
+
+.usa-icon-list__content {
+  font-size: $type-scale;
+  padding-left: $type-scale * $icon-list-icon-padding-left-factor;
+  > *:first-child {
+    margin-top: 0;
+  }
+  > *:last-child {
+    margin-bottom: 0;
+  }
+
+  ul li {
+    list-style-type: disc;
+  }
+}
```

Comment:
We also need a `_usa-icon-list.scss` package and add it to `styles.css` if it's not included in `packages/_usa-icon`.

Reply:
Added